### PR TITLE
Pr to test draft prs again.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Somebody without write/commit/labels access please attempt to set this as ready for review

@Sabiram

### CODE REVIEW

**Review by Tlaltecuhtli — APPROVED** [review]

File: config/admins.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ CitrusGender = Game Master
 NewSta = Game Master
 Expletives = Game Master
 kingofkosmos = Game Master
-MrStonedOne = Lazy Master
```
Comment (MrDoomBringer) [inline config/admins.txt]: bad change

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

**Comment by Ziiro** [comment]
Neat, the icon is different now
chrome_2019-02-15_13-16-55

**Comment by sabiram** [comment]
@MrStonedOne Can't do it

**Comment by ExcessiveUseOfCobblestone** [comment]
I can't undraft the PR how boring

**Pull request closed without merging**